### PR TITLE
Collide utils

### PR DESCRIPTION
**Pull request by antoine4livre**
**TODO:**
- [x] Create collide utils for rectangles and circles
- [x] Adding tests to collide utils
- [x] Write documentation of collide utils

### CODE REVIEW

**Comment by titouanmathis** [comment]
@antoine4livre I rebased your feature on develop to include the recent changes in the folder structure and to avoid conflicts.
Your force pushes undid these changes, you should rebase again in order for the Netlify builds to pass.

**Comment by antoine4livre** [comment]
I think that it is finished @titouanmathis, if you want to review it.

File: packages/js-toolkit/utils/collide/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Inspired by http://www.jeffreythompson.org/collision-detection/
+ */
+import collidePointRect from './collidePointRect.js;
```
Comment (titouanmathis) [inline packages/js-toolkit/utils/collide/index.js:4]: **issue:** There is a missing closing quote here

**Review by titouanmathis — CHANGES_REQUESTED** [review]
Seems all good to me! 

There is one typo that prevents the test from passing, but it should be good afterwards.

**Comment by titouanmathis** [comment]
@antoine4livre Could you update the `tests/utils/index.spec.js` test to include the new `collide` export ?

**Pull request merged**